package.json

```
@@ -22,6 +22,7 @@
   "bugs": {
     "url": "https://github.com/typescript-eslint/typescript-eslint/issues"
   },
+  "homepage": "https://typescript-eslint.io/",
```

This is super nitpicky, but - there's no need for the trailing slash 😄. Most existing files don't use it.  

```suggestion
  "homepage": "https://typescript-eslint.io",
```